src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/WorkbookRepository.kt

```diff
@@ -160,7 +160,7 @@ class WorkbookRepository(
             .filter { localDate -> localDate.value != null }
             .filter { take == selectedTakeRelay.value?.value }
             .map { null }
-            .subscribe { selectedTakeRelay }
```

Comment:
Static type-checking wasn't good enough here, but I'm SO GLAD we're not in a dynamic language.